examples/WebClient/WebClient.ino

```diff
@@ -0,0 +1,167 @@
+/*
+  Web client
+
+  This sketch connects to a website (http://www.google.com)
+  using an Arduino Wiznet Ethernet shield.
+
+  Circuit:
+   Ethernet shield attached to pins 10, 11, 12, 13
+
+  created 18 Dec 2009
+  by David A. Mellis
+  modified 9 Apr 2012
+  by Tom Igoe, based on work by Adrian McEwen
+
+*/
+
+#include <QNEthernet.h>
+
+using namespace qindesign::network;
+
+// if you don't want to use DNS (and reduce your sketch size)
+// use the numeric IP instead of the name for the server:
+char server[] = "arduino.cc";  // also change the Host line in httpRequest()
+//IPAddress server(100,24,172,113);
+
+#define USING_DHCP    true
+
+#if !USING_DHCP
+  // Set the static IP address to use if the DHCP fails to assign
```

Comment:
Same comment as previous example.

Reply:
Same as above. !USING_DHCP, not USING_DHCP